limits.go

```diff
@@ -0,0 +1,75 @@
+package libp2p
+
+import (
+	"github.com/libp2p/go-libp2p/p2p/host/autonat"
+	circuitv1 "github.com/libp2p/go-libp2p/p2p/protocol/circuitv1/relay"
+	circuitv2 "github.com/libp2p/go-libp2p/p2p/protocol/circuitv2/relay"
+	"github.com/libp2p/go-libp2p/p2p/protocol/holepunch"
+	"github.com/libp2p/go-libp2p/p2p/protocol/identify"
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+
+	rcmgr "github.com/libp2p/go-libp2p-resource-manager"
+)
+
+// SetDefaultServiceLimits sets the default limits for bundled libp2p services
```

Comment:
Can we document what these limits are so users don't have to read the source?

Reply:
sure!